fairlearn/__init__.py

```diff
@@ -9,7 +9,7 @@
 # Finesse the version
 _FAIRLEARN_DEV_VERSION_ENV_VAR = "FAIRLEARN_DEV_VERSION"
 
-_base_version = "0.4.4"
+_base_version = "0.4.5"
```

Comment:
Isn't the version mentioned in the README and perhaps other places?

Reply:
I checked. The README definitely has the latest number.

Reply:
Oops. Thanks for the reminder :-/